wcivf/apps/elections/tests/test_postcode_views.py

```diff
@@ -254,8 +254,6 @@ def test_multiple_elections_london(self, mock_response, client):
             election__election_date="2024-05-06",
             election__name="Cities of London and Westminster by-election",
         )
-        parl_london.is_before_registration_deadline = True
-        local_london.is_before_registration_deadline = True
```

Comment:
None of these properties I removed were doing anything. This is not a property of a ballot.